.storybook/mock-data/loan-data-mock.js

```diff
@@ -60,6 +60,7 @@ export default [
 				"__typename": "Image"
 			},
 			"anonymizationLevel": "none",
+			"fullLoanUse": "A loan of $600 helps to purchase heifers  to increase headcount of cattle and sales of organic milk.",
```

Comment:
Looks like double space after "heifers"

Reply:
Yeah, I saw that too, but that's in the original loan use statement so I preserved it.